### ISSUE TITLE
[Question] Error one or more entries with a date in the future

### ISSUE THREAD

**Issue by BoykoAlex** [microsoft/vscode-vsce#797]
While attempting to publish an extension I get the following error:
```
INFO  Publishing 'Pivotal.vscode-bosh v1.42.0'...
 ERROR  One or more entries have a modified date in the future.
```

Can you please explain a bit where this error is coming from? List some possible sources for this error?

**Comment by BoykoAlex** [comment]
NVM :-)

**Comment by lionello** [comment]
@BoykoAlex Could you add some more info as to what fixed the problem for you? The "NVM" isn't very helpful to others with the same problem.

**Comment by lionello** [comment]
In my case, I had to add `.envrc` and `.direnv/` to the `.vscodeignore` file.

First, I used `vsce package` to manual create the `vsix` file and upload it to the marketplace using the browser. Same error. Then, I changed the package file extension to `.zip` so I could inspect the contents with `unzip`. I saw a bunch of files in `.direnv` sub-folder, which are not needed. Once I updated the `.vscodeignore` file, `vsce publish` worked!

**Comment by BoykoAlex** [comment]
@lionello yes indeed :-) My case was rather simple at the end. One of the maven jars that I used in my language server included in the vscode extension had last modified stamp from year 2121 which is far in the future. Therefore `npm` complained about the date in the future. I have switched this maven dependency scope to `test` scope where it really belongs and it fixed the issue for me.
The error was showing for me when i ran `vsce package` locally...
